CAP/gap/MethodRecord.gi

```diff
@@ -3104,3 +3096,20 @@ InstallGlobalFunction( CAP_INTERNAL_ENHANCE_NAME_RECORD,
 end );
 
 CAP_INTERNAL_ENHANCE_NAME_RECORD( CAP_INTERNAL_METHOD_NAME_RECORD );
+
+##
+InstallGlobalFunction( CAP_INTERNAL_REVERSE_LISTS_IN_ARGUMENTS_FOR_OPPOSITE,
```

Comment:
Just a little nitpick: Should this function be in the `MethodRecord.gi` or maybe even in `ToolsForCategories.gi` or so? It does not really matter though, we can always move it.

Reply:
Unfortunately, the function cannot be in `ToolsForCategories.gi` since this file is already loaded in the `init.g` at some very specific place.

Reply:
I see, we also discussed that. Maybe we should have two `tools` files, one that needs to be loaded very early, and one with the rest that can be loaded using the normal scheme

Reply:
Ok, I suggest I merge this PR as it is now, and we resolve the Tools issue in a seperate thread.

Reply:
Yep.